### ISSUE TITLE
0.17.x breaks compilation on linux

### ISSUE THREAD

**Issue by marbemac** [FormidableLabs/radium#654]
Using Meteor, can no longer deploy to linux platforms - get the error:

```
this app is not compatible with architecture 'os.linux.x86_64'
```

To reproduce, pull down meteor, add radium 0.17, and run:

```
meteor build ./build --architecture os.linux.x86_64 --allow-incompatible-update
```
Isolated the issue to radium 0.17 (0.16.x works fine). Could be a change in radium, or radium's dependencies, not sure. Sorry, but this is all of the information that I have!

**Comment by alexlande** [comment]
Added a repro of this issue here: https://github.com/alexlande/radium-meteor-repro

Presumably this is the result of a dependency change in `inline-style-prefixer`/`inline-style-prefix-all`, since there weren't a whole lot of other changes between `0.16.x` and `0.17.0`.

**Comment by alexlande** [comment]
The plot thickens. When I install `0.16.6`, I'm seeing that the `package.json` on the npm package doesn't match the one in the repo. Specifically, the `babel` dependencies are in `devDependencies` in the npm package, when they should be in dependencies. This means that they aren't being installed in the meteor app.

If you manually drop those packages into radium `0.16.6`'s `node_modules` in the meteor repro, the build breaks.

Asked @tptee to confirm this one.

**Comment by alexlande** [comment]
Ah, the babel movement appears to be expected: https://github.com/FormidableLabs/radium/blob/master/package.json#L29-L30

So now the real question is-- why isn't that process working in `0.17.0`?

**Comment by alexlande** [comment]
I just published `0.17.1-test` to test this out, and I have no idea what's going on.

As far as I can tell, the `prepublish` script ran correctly, ending with this:

```
> radium@0.17.1-test deps-fix /Users/alande/Documents/web/radium
> babel-node ./npm-scripts/move-babel-to-dependencies.js

+ radium@0.17.1-test

> radium@0.17.1-test postpublish .
> npm run deps-restore


> radium@0.17.1-test deps-restore /Users/alande/Documents/web/radium
> git checkout package.json
```

But, upon installing `0.17.1-test` and looking at it, the `package.json` was incorrect and the babel plugins are included in `dependencies`.

Running `npm run deps-fix` and `npm run deps-restore` manually is working as expected.

@ianobermiller: Any ideas? I'm stumped on this one. Tempted to run the full prepublish script (including `deps-fix`) manually and then publish. That way we could get `0.17.1` out with this problem resolved (although we would still have the problem that `prepublish` is not working as intended in all cases).

**Comment by alexlande** [comment]
Going to publish `0.17.1` with a fix. Will open a separate ticket to look at the `fix-deps` issue after.

**Comment by alexlande** [comment]
Resolved with `0.17.1`
